python/lsst/ts/xml/data/sal_interfaces/HVAC/HVAC_Events.xml

```diff
@@ -287,12 +269,192 @@
-      <IDL_Type>int</IDL_Type>
+      <Description>Dynalene TMA Alarm state; On (true) or Off (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_logevent_dynTMAalarmCMD</EFDB_Topic>
+    <Description>Dynalene TMA Alarm CMD State.</Description>
+    <item>
+      <EFDB_Name>state</EFDB_Name>
+      <Description>Dynalene TMA Alarm CMD state; On (true) or Off (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_logevent_dynTMAalarmMonitorON</EFDB_Topic>
+    <Description>Dynalene TMA Alarm Monitor ON State.</Description>
+    <item>
+      <EFDB_Name>state</EFDB_Name>
+      <Description>Dynalene TMA Alarm Monitor ON state; On (true) or Off (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_logevent_dynTMAalarmMonitorOFF</EFDB_Topic>
+    <Description>Dynalene TMA Alarm Monitor OFF State.</Description>
+    <item>
+      <EFDB_Name>state</EFDB_Name>
+      <Description>Dynalene TMA Alarm Monitor OFF state; On (true) or Off (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_logevent_dynTAalarm</EFDB_Topic>
+    <Description>Dynalene TA Alarm State.</Description>
+    <item>
+      <EFDB_Name>state</EFDB_Name>
+      <Description>Dynalene TA Alarm state; On (true) or Off (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_logevent_dynTAalarmCMD</EFDB_Topic>
+    <Description>Dynalene TA Alarm CMD State.</Description>
+    <item>
+      <EFDB_Name>state</EFDB_Name>
+      <Description>Dynalene TA Alarm CMD state; On (true) or Off (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_logevent_dynTAalarmMonitorON</EFDB_Topic>
+    <Description>Dynalene TA Alarm Monitor ON State.</Description>
+    <item>
+      <EFDB_Name>state</EFDB_Name>
+      <Description>Dynalene TA Alarm Monitor ON state; On (true) or Off (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_logevent_dynTAalarmMonitorOFF</EFDB_Topic>
```

Comment:
Is this saying that the alarm monitor is turned off? Maybe consider merging these two events together into one event.